src/App/app.scss

```diff
@@ -42,12 +42,16 @@
             overflow: hidden;
             width: 100%;
 
-            margin-bottom: 0;
-            transition: margin-bottom $normal-transition;
+            max-height: $log-viewer-height;
```

Comment:
I assume changing the animated property from `margin-bottom` to `max-height` does change the form of animation from sliding out the log to shrinking it. Is that correct and are you fine with that?

Reply:
It does, but the difference wasn't really perceptible as far as I could tell. But that could just be me, so we can look at improving that later, if this change is in fact noticeable.